recipes/pycaret/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "pycaret" %}
+{% set version = "2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5db9060f9d8ba53dbfb558b41de822c5b4175223c05a3a2c3c41c71346510694
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  # Limitations here because of xgboost package.
+  skip: true  # [win or linux32 or py2k]
```

Comment:
Can this be noarch python? We don't build on linux32, py2k can be solved with `python>=3` below and win probably doesn't have all the deps

Reply:
I added `noarch: python`. I had that originally but the windows build failed so I removed it and added the skip line, which I got from the xgboost package (the dep that doesn't exist on windows)

Reply:
actually nevermind - noarch won't work which selectors. Should I leave noarch in even though the windows version doesn't build?

Reply:
noarch packages only need to build on linux.  It is OK if windows fails.  Please make restore the noarch :wink:

Reply:
Done!